### ISSUE TITLE
Cannot enter folders ending with a '.' (dot)

### ISSUE THREAD

**Issue by darkcharmander** [billziss-gh/sshfs-win#176]
Just as the title says, you cannot enter folders ending with a '.'.
Windows will give you the usual error 'Location not available'.
SMB on Windows suffers from exactly the same problem.

I think it's possible to fix this with a small hack: Make explorer read it as '%2E', an encoded path. Samba is somewhat capable of doing this, and RaiDrive also does this with SFTP.

**Comment by billziss-gh** [comment]
Yes, Windows considers such filenames invalid. See [here](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file).

> Do not end a file or directory name with a space or a period. Although the underlying file system may support such names, the Windows shell and user interface does not. However, it is acceptable to specify a period as the first character of a name. For example, ".temp".

Now SSHFS-Win supports files that contain other invalid characters (e.g. *) by mapping them to a Unicode Private Use Area. Perhaps we can do the same for the case of "trailing dot".

**Comment by darkcharmander** [comment]
I finally managed to build sshfs by using the instructions from #41. Perhaps I will be able to fix this issue myself now. Cannot guarantee anything however.
